security/yubikey-manager/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+name                yubikey-manager
+github.setup        Yubico yubikey-manager 0.6.0 yubikey-manager-
+categories          security python
+platforms           darwin
+license             BSD
+maintainers         {@amake madlon-kay.com:aaron+macports} openmaintainer
+
+description         Tool ("ykman") for managing your YubiKey configuration.
+long_description    ${description}
+
+homepage            https://developers.yubico.com/yubikey-manager/
+
+checksums           rmd160  f4a621975f73ed2ec22f95e5813c5307b269dd6f \
+                    sha256  2a14d0f648f323bfcf97076908d2cb8d7070effb5ff4c8222c44cd687e9c0262
+
+python.default_version 27
```

Comment:
To my understanding, we are now specifying `36` as the "default" `python.default_version` for things that will work under 3.6.

Reply:
Oh, and you might want to specify

`python.versions     27 35 36`

if it will indeed build on those versions.

Reply:
@pmetzger: this is only ever helpful for python modules that can in fact build multiple versions at the same time or for ports that bother implementing their own `+pythonXY` variants. I have absolutely no idea if providing multiple versions here would be of any benefit (it would not work at all). Yes, specifying 36 if it works correctly would be ideal / most future-proof, but specifying multiple versions not so much.

Reply:
@mojca Gotcha.